lib/zendesk_api/resources.rb

```diff
@@ -857,4 +857,16 @@ class Variant < ZendeskAPI::Resource
       has_many Variant
     end
   end
+
+  class PushNotificationDevice < Resource
+    post :destroy_many
+
+    def self.resource_path
+      "push_notification_devices/destroy_many"
+    end
+
+    def self.destroy_many(client, tokens)
+     ZendeskAPI::Collection.new(client, self, "push_notification_devices" => tokens, :verb => :post).fetch
```

Comment:
@steved I don't see any other callas to .fetch here. Am I doing something worng by calling this fetch here? I think I am :smile: